README.md

```diff
@@ -59,7 +59,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+The expression weather = "sunny" assigns the value of "sunny" to the variable "weather". Weather === "sunny" will return a value of "true."
```

Comment:
All true! However, the question was asking what each of those lines evaluated to. 

Line 1, `var weather;`, evaluates to `undefined`.
Line 2, `weather = "sunny"`, evaluates to `"sunny"`, because the `=` operator always evaluates to the value that it's assigning.
Line 3, `weather === "sunny"`, evaluates to `true` because `weather` is equal to `"sunny"`.